README.md

```diff
@@ -182,6 +182,8 @@ Visual regression testing is implemented by [BackstopJS](https://github.com/garr
 ### Setup docker
 
 Install docker on your machine [Download Docker](https://hub.docker.com/search/?type=edition&offering=community&architecture=amd64)
+`see  more detail on how to install docker [here](https://docs.docker.com/docker-for-windows/install/)`
```

Comment:
I think the markdown format is wrong and why do we need a specific link for windows? You can find docker CE installation guide for all the platforms from Download Docker link.

Reply:
https://docs.docker.com/docker-for-windows/install/ 
this is link for documentation not for downloading..